src/site/content/pl/explore/index.md

```diff
@@ -0,0 +1,5 @@
+---
+layout: explore
+title: Badać
```

Comment:
Please replace with "Eksploruj"

Reply:
I have corrected the word for the pl language